utam-core/src/test/java/utam/core/framework/context/MobilePlatformTypeTests.java

```diff
@@ -60,12 +75,42 @@ public void testGetActivePlatformProfile() {
   }
 
   @Test
-  public void testAsProfile() {
-    Profile profile = ANDROID;
+  public void testAsProfileAndroidPhone() {
+    Profile profile = ANDROID_PHONE;
     assertThat(profile.getName(), is(equalTo(PLATFORM_PROFILE_NAME)));
-    assertThat(profile.getValue(), is(equalTo("android")));
-    assertThat(profile.getConfigName(null), is(equalTo("platform_android_config")));
-    assertThat(profile.getConfigName(""), is(equalTo("platform_android_config")));
-    assertThat(profile.getConfigName("my"), is(equalTo("my_platform_android_config")));
+    assertThat(profile.getValue(), is(equalTo("android_phone")));
+    assertThat(profile.getConfigName(null), is(equalTo("platform_android_phone_config")));
+    assertThat(profile.getConfigName(""), is(equalTo("platform_android_phone_config")));
+    assertThat(profile.getConfigName("my"), is(equalTo("my_platform_android_phone_config")));
+  }
+
+  @Test
+  public void testAsProfileAnddroidTablet() {
+    Profile profile = ANDROID_TABLET;
+    assertThat(profile.getName(), is(equalTo(PLATFORM_PROFILE_NAME)));
+    assertThat(profile.getValue(), is(equalTo("android_tablet")));
+    assertThat(profile.getConfigName(null), is(equalTo("platform_android_tablet_config")));
+    assertThat(profile.getConfigName(""), is(equalTo("platform_android_tablet_config")));
+    assertThat(profile.getConfigName("my"), is(equalTo("my_platform_android_tablet_config")));
```

Comment:
pls remove lines 92-93, it's redundant to test on every profile we create
same for the next 2 tests

Reply:
removed